app/javascript/packages/document-capture/components/document-capture-selfie-capture.jsx

```diff
@@ -0,0 +1,59 @@
+import { t } from '@18f/identity-i18n';
+import DocumentSideAcuantCapture from './document-side-acuant-capture';
+import TipList from './tip-list';
+
+/** @typedef {import('@18f/identity-form-steps').FormStepError<*>} FormStepError */
+/** @typedef {import('@18f/identity-form-steps').RegisterFieldCallback} RegisterFieldCallback */
+/** @typedef {import('@18f/identity-form-steps').OnErrorCallback} OnErrorCallback */
+
+/**
+ * @typedef DocumentCaptureSelfieCaptureProps
+ *
+ * @prop {RegisterFieldCallback} registerField
+ * @prop {Blob|string|null|undefined} value
+ * @prop {(nextValues:{[key:string]: Blob|string|null|undefined})=>void} onChange Update values,
+ * merging with existing values.
+ * @prop {FormStepError[]} errors
+ * @prop {OnErrorCallback} onError
+ * @prop {string=} className
+ */
+
+/**
+ * @param {DocumentCaptureSelfieCaptureProps} props Props object.
+ */
+function DocumentCaptureSelfieCapture({
+  registerField,
+  value,
+  onChange,
+  errors,
+  onError,
+  className,
+}) {
+  return (
+    <>
+      <hr className="margin-y-5" />
+      <h2>{`2. ${t('doc_auth.headings.document_capture_subheader_selfie')}`}</h2>
```

Comment:
I haven't tested, but can this be simplified as... ?

```suggestion
      <h2>2. ${t('doc_auth.headings.document_capture_subheader_selfie')}</h2>
```

Reply:
@aduth , that' what I did originally. I can put it back.